### ISSUE TITLE
2025-01-15, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#52]
# Today's Arxiv Papers

(1) [PokerBench: Training Large Language Models to become Professional Poker Players](http://arxiv.org/abs/2501.08328v1), 2025-01-14

(2) [Omni-RGPT: Unifying Image and Video Region-level Understanding via Token Marks](http://arxiv.org/abs/2501.08326v1), 2025-01-14

(3) [ADAM-1: AI and Bioinformatics for Alzheimer's Detection and Microbiome-Clinical Data Integrations](http://arxiv.org/abs/2501.08324v1), 2025-01-14

(4) [Exploring Robustness of Multilingual LLMs on Real-World Noisy Data](http://arxiv.org/abs/2501.08322v1), 2025-01-14

(5) [Enhancing Automated Interpretability with Output-Centric Feature Descriptions](http://arxiv.org/abs/2501.08319v1), 2025-01-14

(6) [HALoGEN: Fantastic LLM Hallucinations and Where to Find Them](http://arxiv.org/abs/2501.08292v1), 2025-01-14

(7) [AfriHate: A Multilingual Collection of Hate Speech and Abusive Language Datasets for African Languages](http://arxiv.org/abs/2501.08284v1), 2025-01-14

(8) [LLaVA-ST: A Multimodal Large Language Model for Fine-Grained Spatial-Temporal Understanding](http://arxiv.org/abs/2501.08282v1), 2025-01-14

(9) [Exploring Robustness of LLMs to Sociodemographically-Conditioned Paraphrasing](http://arxiv.org/abs/2501.08276v1), 2025-01-14

(10) [Addressing the sustainable AI trilemma: a case study on LLM agents and RAG](http://arxiv.org/abs/2501.08262v1), 2025-01-14

(11) [Eliciting In-context Retrieval and Reasoning for Long-context Large Language Models](http://arxiv.org/abs/2501.08248v1), 2025-01-14

(12) [Text-Diffusion Red-Teaming of Large Language Models: Unveiling Harmful Behaviors with Proximity Constraints](http://arxiv.org/abs/2501.08246v1), 2025-01-14

(13) [Engineering LLM Powered Multi-agent Framework for Autonomous CloudOps](http://arxiv.org/abs/2501.08243v1), 2025-01-14

(14) [Investigating Energy Efficiency and Performance Trade-offs in LLM Inference Across Tasks and DVFS Settings](http://arxiv.org/abs/2501.08219v1), 2025-01-14

(15) [ASTRID -- An Automated and Scalable TRIaD for the Evaluation of RAG-based Clinical Question Answering Systems](http://arxiv.org/abs/2501.08208v1), 2025-01-14

(16) [ArithmAttack: Evaluating Robustness of LLMs to Noisy Context in Math Problem Solving](http://arxiv.org/abs/2501.08203v1), 2025-01-14

(17) [CWEval: Outcome-driven Evaluation on Functionality and Security of LLM Code Generation](http://arxiv.org/abs/2501.08200v1), 2025-01-14

(18) [OpenCSG Chinese Corpus: A Series of High-quality Chinese Datasets for LLM Training](http://arxiv.org/abs/2501.08197v1), 2025-01-14

(19) [PRESERVE: Prefetching Model Weights and KV-Cache in Distributed LLM Serving](http://arxiv.org/abs/2501.08192v1), 2025-01-14

(20) [A Multi-Modal AI Copilot for Single-Cell Analysis with Instruction Following](http://arxiv.org/abs/2501.08187v1), 2025-01-14

(21) [Potential and Perils of Large Language Models as Judges of Unstructured Textual Data](http://arxiv.org/abs/2501.08167v1), 2025-01-14

(22) [I Can Find You in Seconds! Leveraging Large Language Models for Code Authorship Attribution](http://arxiv.org/abs/2501.08165v1), 2025-01-14

(23) [Refusal Behavior in Large Language Models: A Nonlinear Perspective](http://arxiv.org/abs/2501.08145v1), 2025-01-14

(24) [Consistency of Responses and Continuations Generated by Large Language Models on Social Media](http://arxiv.org/abs/2501.08102v1), 2025-01-14

(25) [Hierarchical Autoscaling for Large Language Model Serving with Chiron](http://arxiv.org/abs/2501.08090v1), 2025-01-14

(26) [CuAsmRL: Optimizing GPU SASS Schedules via Deep Reinforcement Learning](http://arxiv.org/abs/2501.08071v1), 2025-01-14

(27) [A Roadmap to Guide the Integration of LLMs in Hierarchical Planning](http://arxiv.org/abs/2501.08068v1), 2025-01-14

(28) [Exploring Narrative Clustering in Large Language Models: A Layerwise Analysis of BERT](http://arxiv.org/abs/2501.08053v1), 2025-01-14

(29) [TriAdaptLoRA: Brain-Inspired Triangular Adaptive Low-Rank Adaptation for Parameter-Efficient Fine-Tuning](http://arxiv.org/abs/2501.08008v1), 2025-01-14

(30) [LLM-Ehnanced Holonic Architecture for Ad-Hoc Scalable SoS](http://arxiv.org/abs/2501.07992v1), 2025-01-14

(31) [Facial Dynamics in Video: Instruction Tuning for Improved Facial Expression Perception and Contextual Awareness](http://arxiv.org/abs/2501.07978v1), 2025-01-14

(32) [Zero-shot Video Moment Retrieval via Off-the-shelf Multimodal Large Language Models](http://arxiv.org/abs/2501.07972v1), 2025-01-14

(33) [Self-Instruct Few-Shot Jailbreaking: Decompose the Attack into Pattern and Behavior Learning](http://arxiv.org/abs/2501.07959v1), 2025-01-14

(34) [Advice for Diabetes Self-Management by ChatGPT Models: Challenges and Recommendations](http://arxiv.org/abs/2501.07931v1), 2025-01-14

(35) [Gandalf the Red: Adaptive Security for LLMs](http://arxiv.org/abs/2501.07927v1), 2025-01-14

(36) [Large Language Model Interface for Home Energy Management Systems](http://arxiv.org/abs/2501.07919v1), 2025-01-14

(37) [Leveraging Metamemory Mechanisms for Enhanced Data-Free Code Generation in LLMs](http://arxiv.org/abs/2501.07892v1), 2025-01-14

(38) [ReARTeR: Retrieval-Augmented Reasoning with Trustworthy Process Rewarding](http://arxiv.org/abs/2501.07861v1), 2025-01-14

(39) [Hierarchical Repository-Level Code Summarization for Business Applications Using Local LLMs](http://arxiv.org/abs/2501.07857v1), 2025-01-14

(40) [Optimizing Language Models for Grammatical Acceptability: A Comparative Study of Fine-Tuning Techniques](http://arxiv.org/abs/2501.07853v1), 2025-01-14

(41) [Unveiling Provider Bias in Large Language Models for Code Generation](http://arxiv.org/abs/2501.07849v1), 2025-01-14

(42) [Reasoning with Graphs: Structuring Implicit Knowledge to Enhance LLMs Reasoning](http://arxiv.org/abs/2501.07845v1), 2025-01-14

(43) [A Driver Advisory System Based on Large Language Model for High-speed Train](http://arxiv.org/abs/2501.07837v1), 2025-01-14

(44) [Flow: A Modular Approach to Automated Agentic Workflow Generation](http://arxiv.org/abs/2501.07834v1), 2025-01-14

(45) [Real-time Verification and Refinement of Language Model Text Generation](http://arxiv.org/abs/2501.07824v1), 2025-01-14

(46) [3UR-LLM: An End-to-End Multimodal Large Language Model for 3D Scene Understanding](http://arxiv.org/abs/2501.07819v1), 2025-01-14

(47) [A Multi-Encoder Frozen-Decoder Approach for Fine-Tuning Large Language Models](http://arxiv.org/abs/2501.07818v1), 2025-01-14

(48) [Agent-Centric Projection of Prompting Techniques and Implications for Synthetic Training Data for Large Language Models](http://arxiv.org/abs/2501.07815v1), 2025-01-14

(49) [Talk to Right Specialists: Routing and Planning in Multi-agent System for Question Answering](http://arxiv.org/abs/2501.07813v1), 2025-01-14

(50) [CodeCoR: An LLM-Based Self-Reflective Multi-Agent Framework for Code Generation](http://arxiv.org/abs/2501.07811v1), 2025-01-14

(51) [Visual Language Models as Operator Agents in the Space Domain](http://dx.doi.org/10.2514/6.2025-1543), 2025-01-14

(52) [Parameter-Inverted Image Pyramid Networks for Visual Perception and Multimodal Understanding](http://arxiv.org/abs/2501.07783v1), 2025-01-14

(53) [Large Language Models for Knowledge Graph Embedding Techniques, Methods, and Challenges: A Survey](http://arxiv.org/abs/2501.07766v1), 2025-01-14

(54) [Advancing Student Writing Through Automated Syntax Feedback](http://arxiv.org/abs/2501.07740v1), 2025-01-13

(55) [LLMic: Romanian Foundation Language Model](http://arxiv.org/abs/2501.07721v1), 2025-01-13

(56) [Entailed Between the Lines: Incorporating Implication into NLI](http://arxiv.org/abs/2501.07719v1), 2025-01-13

(57) [Benchmarking Abstractive Summarisation: A Dataset of Human-authored Summaries of Norwegian News Articles](http://arxiv.org/abs/2501.07718v1), 2025-01-13

(58) [Constructing Set-Compositional and Negated Representations for First-Stage Ranking](http://arxiv.org/abs/2501.07679v1), 2025-01-13

(59) [CDS: Data Synthesis Method Guided by Cognitive Diagnosis Theory](http://arxiv.org/abs/2501.07674v1), 2025-01-13

(60) [Enhancing Talent Employment Insights Through Feature Extraction with LLM Finetuning](http://arxiv.org/abs/2501.07663v1), 2025-01-13

(61) [Large Language Models for Interpretable Mental Health Diagnosis](http://arxiv.org/abs/2501.07653v1), 2025-01-13

(62) [BlobGEN-Vid: Compositional Text-to-Video Generation with Blob Video Representations](http://arxiv.org/abs/2501.07647v1), 2025-01-13

(63) [GPT as a Monte Carlo Language Tree: A Probabilistic Perspective](http://arxiv.org/abs/2501.07641v1), 2025-01-13

(64) [SafePowerGraph-LLM: Novel Power Grid Graph Embedding and Optimization with Large Language Models](http://arxiv.org/abs/2501.07639v1), 2025-01-13

(65) [WebWalker: Benchmarking LLMs in Web Traversal](http://arxiv.org/abs/2501.07572v2), 2025-01-13

(66) [SST-EM: Advanced Metrics for Evaluating Semantic, Spatial and Temporal Aspects in Video Editing](http://arxiv.org/abs/2501.07554v1), 2025-01-13

(67) [Imagine while Reasoning in Space: Multimodal Visualization-of-Thought](http://arxiv.org/abs/2501.07542v1), 2025-01-13

(68) [ML Mule: Mobile-Driven Context-Aware Collaborative Learning](http://arxiv.org/abs/2501.07536v1), 2025-01-13

(69) [Investigating Large Language Models in Inferring Personality Traits from User Conversations](http://arxiv.org/abs/2501.07532v1), 2025-01-13

(70) [Evaluating Agent-based Program Repair at Google](http://arxiv.org/abs/2501.07531v1), 2025-01-13

(71) [RadAlign: Advancing Radiology Report Generation with Vision-Language Concept Alignment](http://arxiv.org/abs/2501.07525v1), 2025-01-13

(72) [Parallel Key-Value Cache Fusion for Position Invariant RAG](http://arxiv.org/abs/2501.07523v1), 2025-01-13

(73) [Exploring and Mitigating Adversarial Manipulation of Voting-Based Leaderboards](http://arxiv.org/abs/2501.07493v1), 2025-01-13

(74) [TiEBe: A Benchmark for Assessing the Current Knowledge of Large Language Models](http://arxiv.org/abs/2501.07482v1), 2025-01-13

(75) [A Survey of Embodied AI in Healthcare: Techniques, Applications, and Opportunities](http://arxiv.org/abs/2501.07468v1), 2025-01-13

(76) [Understanding and Benchmarking Artificial Intelligence: OpenAI's o3 Is Not AGI](http://arxiv.org/abs/2501.07458v1), 2025-01-13

(77) [Enhancing LLM's Ability to Generate More Repository-Aware Unit Tests Through Precise Contextual Information Injection](http://arxiv.org/abs/2501.07425v1), 2025-01-13

(78) [Initial Findings on Sensor based Open Vocabulary Activity Recognition via Text Embedding Inversion](http://arxiv.org/abs/2501.07408v1), 2025-01-13

(79) [Emergent effects of scaling on the functional hierarchies within large language models](http://arxiv.org/abs/2501.07359v1), 2025-01-13

(80) [Occamy: A 432-Core Dual-Chiplet Dual-HBM2E 768-DP-GFLOP/s RISC-V System for 8-to-64-bit Dense and Sparse Computing in 12nm FinFET](http://dx.doi.org/10.1109/JSSC.2025.3529249), 2025-01-13

(81) [Foundation Models at Work: Fine-Tuning for Fairness in Algorithmic Hiring](http://arxiv.org/abs/2501.07324v1), 2025-01-13

(82) [FinerWeb-10BT: Refining Web Data with LLM-Based Line-Level Filtering](http://arxiv.org/abs/2501.07314v1), 2025-01-13

(83) [The Lessons of Developing Process Reward Models in Mathematical Reasoning](http://arxiv.org/abs/2501.07301v1), 2025-01-13

(84) [GestLLM: Advanced Hand Gesture Interpretation via Large Language Models for Human-Robot Interaction](http://arxiv.org/abs/2501.07295v1), 2025-01-13

(85) [LLM-Net: Democratizing LLMs-as-a-Service through Blockchain-based Expert Networks](http://arxiv.org/abs/2501.07288v1), 2025-01-13

(86) [Lifelong Learning of Large Language Model based Agents: A Roadmap](http://arxiv.org/abs/2501.07278v1), 2025-01-13

(87) [Bridging Smart Meter Gaps: A Benchmark of Statistical, Machine Learning and Time Series Foundation Models for Data Imputation](http://arxiv.org/abs/2501.07276v1), 2025-01-13

(88) [Transforming Role Classification in Scientific Teams Using LLMs and Advanced Predictive Analytics](http://arxiv.org/abs/2501.07267v1), 2025-01-13

(89) [Large Language Models: New Opportunities for Access to Science](http://arxiv.org/abs/2501.07250v1), 2025-01-13

(90) [Lessons From Red Teaming 100 Generative AI Products](http://arxiv.org/abs/2501.07238v1), 2025-01-13

(91) [Breaking Memory Limits: Gradient Wavelet Transform Enhances LLMs Training](http://arxiv.org/abs/2501.07237v1), 2025-01-13

(92) [Touched by ChatGPT: Using an LLM to Drive Affective Tactile Interaction](http://arxiv.org/abs/2501.07224v1), 2025-01-13

(93) [FaceOracle: Chat with a Face Image Oracle](http://arxiv.org/abs/2501.07202v1), 2025-01-13

(94) [Pre-Trained Large Language Model Based Remaining Useful Life Transfer Prediction of Bearing](http://arxiv.org/abs/2501.07191v1), 2025-01-13

(95) [Unveiling Code Clone Patterns in Open Source VR Software: An Empirical Study](http://arxiv.org/abs/2501.07165v1), 2025-01-13

(96) [FlexQuant: Elastic Quantization Framework for Locally Hosted LLM on Edge Devices](http://arxiv.org/abs/2501.07139v1), 2025-01-13

(97) [LLM360 K2: Scaling Up 360-Open-Source Large Language Models](http://arxiv.org/abs/2501.07124v1), 2025-01-13

(98) [How GPT learns layer by layer](http://arxiv.org/abs/2501.07108v1), 2025-01-13

(99) [ADKGD: Anomaly Detection in Knowledge Graphs with Dual-Channel Training](http://arxiv.org/abs/2501.07078v1), 2025-01-13

(100) [Value Compass Leaderboard: A Platform for Fundamental and Validated Evaluation of LLMs Values](http://arxiv.org/abs/2501.07071v1), 2025-01-13
